shared/middleware/files_api.rb

```diff
@@ -723,8 +729,11 @@ def files_put_file(encrypted_channel_id, filename, body)
     return {filesVersionId: ""}.to_json if manifest_result[:status] == 'NOT_FOUND'
     manifest = JSON.load manifest_result[:body]
 
+    storage_apps = StorageApps.new(get_storage_id)
+    abuse_score = storage_apps.get_abuse(encrypted_channel_id)
+
     # overwrite the manifest file with an empty list
-    response = bucket.create_or_replace(encrypted_channel_id, FileBucket::MANIFEST_FILENAME, [].to_json, params['files-version'])
+    response = bucket.create_or_replace(encrypted_channel_id, FileBucket::MANIFEST_FILENAME, [].to_json, params['files-version'], abuse_score)
```

Comment:
Yuck... we probably shouldn't be tracking abuse scores on the manifest file, but it looks like we are incrementing it in other places: https://github.com/code-dot-org/code-dot-org/blob/8a861dce2c4e71955fdba73d5a92c7b96fad333f/shared/middleware/files_api.rb#L505-L512 

so... this does seem like the right thing for you to do right now. 👍